flake.nix

```diff
@@ -121,7 +121,38 @@
             baseUrl = "https://github.com/hercules-ci/flake-parts/blob/main${sourceSubpath}";
             getModules = f: [ f.flakeModules.easyOverlay ];
             intro = ''
-              Derives a default overlay from `perSystem.packages`.
+              ## WARNING
+
+              This module does NOT make _consuming_ an overlay easy. This module is intended for _creating_ overlays.
+              While it is possible to consume the overlay created by this module using the `final` module argument, this is somewhat unconventional. Instead:
+
+              - _Avoid_ overlays. Many flakes can do without them.
+              - Initialize `pkgs` yourself:
+                ```
+                perSystem = { system, ... }: {
+                  _module.args.pkgs = import inputs.nixpkgs {
+                    inherit system;
+                    overlays = [ (prev: final: {
```

Comment:
Fixed in fa2db0b56a9473db51db15b2e2d003d45f167f84.
Should be

    final: prev: {